### ISSUE TITLE
NVDA install failed.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4118]
**Reported by vgjh2005 on 2014-05-04 07:56**
Hi:
I tried to install NVDA_next10562. But it couldn't be finished.
I also search by entering NVDA to find all files when NVDA were installed.
But the result is the same.
I hope to  Add a option to clear all profile when uninstalling.
The log is below:
**main**.main:
slave error
Traceback (most recent call last):
  File "nvda_slave.pyw", line 29, in main
  File "installer.pyc", line 347, in install
  File "installer.pyc", line 215, in unregisterInstallation
  File "easeOfAccess.pyc", line 75, in setAutoStart
AttributeError: 'list' object has no attribute 'split'
ERROR - gui.installerGui.doInstall (15:43:12):
Installation failed: 1
Thanks!

**Comment by leonardder** [comment]
Seems like a one time occurrence, given the abandoned state of this ticket. An option to clear the profile is suggested in #6869.
